jerry-port/default/default-debugger.c

```diff
@@ -28,7 +28,7 @@ void jerry_port_sleep (uint32_t sleep_time)
 #ifdef HAVE_TIME_H
   nanosleep (&(const struct timespec)
   {
-    sleep_time / 1000, (sleep_time % 1000) * 1000000L /* Seconds, nanoseconds */
+    (long int) sleep_time / 1000, ((long int) sleep_time % 1000) * 1000000L /* Seconds, nanoseconds */
```

Comment:
The type of the first member in timespec is `time_t`. It would be nicer to cast `sleep_time` into it.